src/query/query_dump.c

```diff
@@ -807,7 +807,7 @@ qdump_print_list_id (QFILE_LIST_ID * list_id_p)
       return false;
     }
 
-  fprintf (foutput, ")(tuple_cnt:%d)", list_id_p->tuple_cnt);
+  fprintf (foutput, ")(tuple_cnt:%lld)", list_id_p->tuple_cnt);
```

Comment:
How about explicitly type casting to (long long) when outputting to %lld?

Reply:
I slip it, thank you.